kube_metrics_server/datadog_checks/kube_metrics_server/kube_metrics_server.py

```diff
@@ -7,17 +7,22 @@
 COUNTERS = {
     'authenticated_user_requests': 'authenticated_user.requests',
     'metrics_server_kubelet_summary_scrapes_total': 'kubelet_summary_scrapes_total',
+    'metrics_server_kubelet_request_total': 'kubelet_summary_scrapes_total'
 }
 
 HISTOGRAMS = {
     'metrics_server_manager_tick_duration_seconds': 'manager_tick_duration',
     'metrics_server_scraper_duration_seconds': 'scraper_duration',
 }
 
-SUMMARIES = {'metrics_server_kubelet_summary_request_duration_seconds': 'kubelet_summary_request_duration'}
+SUMMARIES = {
+    'metrics_server_kubelet_summary_request_duration_seconds': 'kubelet_summary_request_duration',
+    'metrics_server_kubelet_request_duration_seconds': 'kubelet_summary_request_duration'
```

Comment:
```suggestion
    'metrics_server_kubelet_request_duration_seconds': 'kubelet_summary_request_duration',
```

Reply:
oh ok that's interesting. i didn't catch the trailing commas after the last element